### ISSUE TITLE
x86_64 fails with the dl.heroesofnewerth path

### ISSUE THREAD

**Issue by mrhappyasthma** [silentbugs/hon-client-scraper#1]
It fails for me for both the manifest and file downloads.

If I use cdn.naeu.patch.heroesofnewerth prefix for the base_url then it appears to work (at least it's downloading files, the size is like 6 gigs so we'll see if it succeeds without any timeouts).

**Comment by mrhappyasthma** [comment]
I believe a quick fix is something like:

```
# main.py line 35
base_url = 'https://cdn.naeu.patch.heroesofnewerth.com/' if self.arch == 'x86_64' else 'http://dl.heroesofnewerth.com/'
```

Then reference that `base_url` in lines `37` and `46`.

**Comment by silentbugs** [comment]
This has been recently implemented and fixed by issue #10. The patcher response is being parsed (poorly, but gets the job done for now) and the url is extracted from it, so we no longer need to hardcode them.
